static/styles/_base/_tables.scss

```diff
@@ -13,9 +15,8 @@ table {
   }
 
   th {
-    padding: 1rem;
-    height: 4rem; 
-    border-bottom: 1px solid $light-gray;
+    padding: 1rem !important;
```

Comment:
Ugh I know. These table styles are all to override the datatables css, which is hyper-specific and includes lots of `!important`s . Is there a smarter way to modify the styles of a node module that I'm missing?